scripts/base/frameworks/cluster/main.zeek

```diff
@@ -280,6 +280,14 @@ export {
 	## Returns: a topic string that may used to send a message exclusively to
 	##          a given cluster node.
 	global nodeid_topic: function(id: string): string;
+
+	## Untyped remote event.
+	type Event: record {
+		## The function to be invoked as an event on the remote side.
```

Comment:
Would it be better here to say "the event handler", even if it's technically a function down in the guts of the script layer? I'm flagging this because for script developers functions, hooks, and events are generally distinct entities.